tools/osbuilder/rootfs-builder/cbl-mariner/config.sh

```diff
@@ -5,6 +5,6 @@
 OS_NAME=cbl-mariner
 OS_VERSION=${OS_VERSION:-2.0}
 LIBC="gnu"
-PACKAGES="core-packages-base-image ca-certificates"
+PACKAGES="kata-packages-uvm"
```

Comment:
If it works, it works, so nothing much to review here.

But I'm curious on what the kata-packages-uvm brings in. :-)

Reply:
Adapted PR description and opening PR. Resolving this comment for now. Thanks for taking an early peek :)